prefq/server.py

```diff
@@ -46,25 +46,22 @@ def index():
 def load_web_interface():
     """Send HTML interface to Feedback Client"""
 
-    # should be replaced by flask session object
-    # pylint: disable=global-statement
-    global videos_rendered
-
-    available_videos = len(filenames_array) - videos_rendered
-    is_video_available = available_videos > 0 and len(filenames_array) != 0
+    is_video_available = 1 if n_pending_queries > 0 else 0
 
     print("\n\nServer: Starting load_web_interface() [...] ")
-    print("Server: Evaluated Videos: " + str(videos_rendered))
-    print("Server: Available Videos: " + str(available_videos))
+    print("Server: Pending Queries: " + str(n_pending_queries))
```

Comment:
the actual purpose of this variable is to communicate the amount of pending queries to the server, before all videos have been sent. In the case of the imitation example, 20 video pairs are sent one at a time, and a feedback client could evaluate all pending_queries from the queue, while the feedback client is still sending videos.

However, eversince the query client does not request the evaluated feedback, before having sent all videos, this could be remodeled in order to get rid of the variable.
```
@app.route("/videos", methods=["POST"])
def receive_videos():
    """Receive new queries from a Query Client."""

    print("\n\nServer: Starting receive_videos() [...]")

    # should be replaced by flask session object
    # pylint: disable=global-statement
    global n_pending_queries

    if n_pending_queries == 0:
        data = flask.request.json
        n_pending_queries = data["n_pending_queries"]
        print("Server: Pending Queries: " + str(n_pending_queries))
        return "Server: [...] Terminating receive_videos()"
```